playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -281,7 +281,7 @@
     - name: Upgrade node packages
       command: yum update -y {{ openshift.common.service_type }}-node{{ openshift_version }}
     - name: Restart node services
-      service: name="{{ openshift.common.service_type }}-node" state=restarted
+      service: name="{{ openshift.common.service_type }}-node" enabled=yes state=restarted
```

Comment:
Same as the master comment above, we should have a separate step checking for started/enabled vs the restart step.

I'm thinking the restart should be handled by a handler as well.

We may also want to check to see if a systemctl daemon-reload is needed to purge the old unit files from the systemd running configuration.